packages/cli/README.md

```diff
@@ -280,6 +280,11 @@ COMMON FLAGS
 DESCRIPTION
   get a specific app or a list of apps
 
+  For API information, see:
+
+  https://developer.smartthings.com/docs/api/public/#operation/listApps,
+  https://developer.smartthings.com/docs/api/public/#operation/getApp
```

Comment:
I would have rather this been a bulleted list but oclif once again is kind of getting in the way with its automatic formatting.